config/config.meta.yaml

```
@@ -146,14 +147,25 @@ clustering:
   temporal:
     resolution_sector: 3H
 
-# procurement strategies
+# ================ Procurment Metastudy ================
 procurement:
+  year: 2030
+  strategy: vol-match # name of procurement strategy "vol-match"
+  scope: "all" # "node", "country", "all"
+  penetration: 100
```

Don't forget to add the changes in the configs and scenarios

```suggestion
  energy_matching: 100
```